### PR TITLE
Update Commit

### PR DESCRIPTION
**Pull request by Janani-m17**

### CODE REVIEW

File: src/main/java/org/example/Main.java [hunk 1]
```diff
@@ -4,15 +4,14 @@
 import java.util.Scanner;
 
 public class Main {
-    public static void main(String[] args) {
-        Scanner s = new Scanner(System.in);
 
+    private static Scanner s = new Scanner(System.in);
```
Comment (Hari-Nikesh-R) [inline src/main/java/org/example/Main.java]: Keep it as proper naming instead of s

File: src/main/java/org/example/Main.java [hunk 2]
```diff
@@ -26,27 +25,7 @@ public static void main(String[] args) {
...
-                            System.out.println("Error: Division by zero.");
-                        }
-                        break;
-                    default:
-                        System.out.println("Enter a valid operation number.");
-                }
+                performOperation(operation,num1,num2);
 
                 System.out.println("Do you want to continue? (yes/no): ");
                 String yesno = s.next().toLowerCase();
```
Comment (Hari-Nikesh-R) [inline src/main/java/org/example/Main.java]: Close the scanner object for better practice
User s.close() for closing it

**Review by Hari-Nikesh-R — CHANGES_REQUESTED** [review]

File: src/main/java/org/example/Main.java (hunk 1 above)
Comment (Janani-m17) [reply src/main/java/org/example/Main.java]: Done anna

File: src/main/java/org/example/Main.java (hunk 2 above)
Comment (Janani-m17) [reply src/main/java/org/example/Main.java]: added anna

**Pull request merged**